### PR TITLE
Additional web-socket functionality

### PR DESCRIPTION
**Pull request by balbazauras**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -17,13 +16,30 @@ def sensor_data(request):
...
         serializer = DataEntrySerializer(data=request.data)
         if serializer.is_valid():
+
             serializer.save()
             return Response(serializer.data)
         return Response(serializer.errors)
+
+
+@api_view(['GET'])
+def get_all_sensor_data(request):
```
Comment (daiarn) [inline api/views.py:33]: it is always `request.method == 'GET'` because of `api_view(['GET'])` decorator

File: mysite/.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
+SECRET_KEY=django-insecure-(h)37e&+=8x%lrbo$@gs2n_a1zot-qt-8j13_ft42=k!0=_s&i
```
Comment (daiarn) [inline mysite/.env:1]: why this file is here? :)

File: static/js/sensor_connection.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  waitForSocketConnection(socket);
+  socket.onmessage = function (event) {
+    processCommand(event);
+  };
+}
+
+function processCommand(event) {
+  let obj = JSON.parse(event.data);
+  document.getElementById("value").innerHTML = obj.value;
+  console.log(obj.value);
```
Comment (daiarn) [inline static/js/sensor_connection.js:44]: do you need console.log here?

**Review by daiarn — APPROVED** [review]
Pls fix everything rebase it and finally merge it to main. Then start working with branches from development and raise PR's to merge into development

File: mysite/.env (hunk 2 above)
Comment (balbazauras) [reply mysite/.env:1]: Created env file before adding it to .gitignore and then forgot to rm it from git :(

**Pull request merged**